Automattic/vip-go-mu-plugins
# [NO REVIEW REQUIRED AT THIS TIME, DO NOT MERGE] Moving vip-dashboard repository to vip-go-mu-plugins

## Description

Moving vip-dashboard repository files to vip-go-mu-plugins repository. Issue request #90.

https://github.com/Automattic/vip-dashboard/issues/90

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Go to `wp-admin` > `Tools` > `Bakery`
1. Click on "Bake Cookies" button.
1. Verify cookies are delicious.
